### DISCUSSION TITLE
Is it possible to store a globally shared bookmark?

### DISCUSSION THREAD

**Discussion by pinkfrog9** [alefragnani/vscode-bookmarks#568] [Q&A] [answered]
Is it possible to store a globally shared bookmark across all vscode projects?

**Comment by alefragnani** [comment, marked as answer]
Hi @introom ,

No, it is not. Bookmarks are _per session/workspace_.

But, there is an open issue #287 asking for this. Feel free to upvote, and this could be available in the future.

Hope this helps
